src/NuGet.Clients/NuGet.CommandLine/NuGet.CommandLine.nuspec

```diff
@@ -5,10 +5,11 @@
     <version>4.6.0</version>
     <authors>Microsoft</authors>
     <owners>Microsoft</owners>
-    <licenseUrl>https://raw.githubusercontent.com/NuGet/NuGet.Client/dev/LICENSE.txt</licenseUrl>
-    <projectUrl>https://github.com/NuGet/NuGet.Client</projectUrl>
-    <copyright>&#169; .NET Foundation. All rights reserved.</copyright>
-    <description>NuGet Command Line Tool</description>
+    <licenseUrl>https://aka.ms/nugetlicense</licenseUrl>
+    <projectUrl>https://aka.ms/nugetprj</projectUrl>
+    <iconUrl>https://raw.githubusercontent.com/NuGet/Media/master/Images/MainLogo/256x256/nuget_256.png</iconUrl>
```

Comment:
Can this be an aka.ms link? Also, I'd consider using a specific commit hash instead of the master branch.

Reply:
I copied the 4.9 branch metadata into the 4.7 branch, so there's not much benefit to making this change here. But it does sound like a good idea for the dev branch so future packages use it.